packages/x-data-grid/src/hooks/features/virtualization/useGridVirtualization.tsx

```diff
@@ -61,9 +67,20 @@ export function useGridVirtualization(
     }));
   };
 
+  const setRowVirtualization = (enabled: boolean) => {
+    apiRef.current.setState((state) => ({
+      ...state,
+      virtualization: {
+        ...state.virtualization,
+        enabledForRows: enabled,
+      },
+    }));
+  };
+
   const api = {
     unstable_setVirtualization: setVirtualization,
     unstable_setColumnVirtualization: setColumnVirtualization,
+    unstable_setRowVirtualization: setRowVirtualization,
```

Comment:
Do we have a use case for it in the userland? If not, could we keep it private for now?

Reply:
I wouldn't add it then, and I would keep the selector private. I'm already unhappy about having needed to add `unstable_setColumnVirtualization`, felt hacky.

Reply:
Makes sense - removed it https://github.com/mui/mui-x/pull/14235/commits/9b61d4258bb44bba09624323e83f480bad4ee9f2